dev/com.ibm.ws.kernel.metatype.helper/resources/OSGI-INF/l10n/fileset.properties

```diff
@@ -31,4 +31,4 @@ fileset.attr.excludes.name=Excludes pattern
 fileset.attr.excludes.desc=The comma or space separated list of file name patterns to exclude from the search results, by default no files are excluded.
 #monitor
 fileset.attr.scan.name=Scan interval
-fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default.
+fileset.attr.scan.desc=Scanning interval to check the fileset for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default. Note: the contents of the individual files are not scanned. Rather, the fileset itself is scanned to determine if files are added or removed from the set.
```

Comment:
Replace"fileset" with "file set".
Consider the text as
"Scanning interval to check the file set for changes as a long with a time unit suffix h-hour, m-minute, s-second, ms-millisecond (e.g. 2ms or 5s). Disabled (scanInterval=0) by default. Note: the contents of the individual files are not scanned. Rather, the fileset itself is scanned to determine if files are added or removed from the set."